src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -0,0 +1,318 @@
+// After check if any update action is present, the legacy entry can be updated using SphereClient instance.
+if (!updateActions.isEmpty()) {
+    sphereClient.execute(InventoryEntryUpdateCommand.of(inventoryEntry, updateActions));
+}
+````
+
+## How does it work?
+
+The inventory sync uses the `sku` and `supplyChannel` key to match new inventory entries to existing ones.
+1. If an inventory entry exists with the same `sku` and a reference to a `supplyChannel` with the same key, it means that
+this inventory entry already exists on the CTP project. Therefore, the tool calculates update actions that should be
+done to update the old inventory entry with the new inventory entry's fields. Only if there are update actions needed,
+they will be issued to the CTP platform.
+2. If no matching inventory entry with the same `sku` and a reference to a `supplyChannel` with the same key is found,
+the tool will create a new one.
+
+The sync, however, will never delete an inventory entry.
+ 
+## FAQ
+#### What does the number of processed inventories actually refer to in the statistics of the sync process?
+It refers to the total number of inventories input to the sync. Under all the following cases an inventory is to be
+counted as processed:
+- new inventory caused the old one to be updated.
+- new inventory was created.
+- new inventory was the same as the old one and requires no action.
+- new inventory failed to process.
+
+The only case where an inventory would not be processed is if the entire sync process fails and stops, before reaching
+this inventory in the input list.
+
+#### Why is it important to provide extended supply channel references in entries from input lists of sync process?
+In CTP inventory allows you to track stock quantity per SKU and optionally per supply channel, what makes that
+both `sku` and `supplyChannel` key are used to inventory entry distinction. Distinction of inventory entries is then
+necessary for comparision of new and old resources during sync process. In both the [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk)
+[InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java)
+and the [InventoryEntry](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntry.java)
+`sku` is provided directly as a `String` field but supply channel key is not. Supply channel is kept as a
+[JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk)
+[Reference](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java)
+what makes the `supplyChannel` key [can't be simply extracted from this reference](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java#L98).
```

Comment:
An inventoryEntry just has a reference to the channel, not the key of the channel (unless [reference expanded](https://dev.commercetools.com/http-api.html#reference-expansion)).